contents/teams/surveys/objectives.mdx

```diff
@@ -1,37 +1,62 @@
 
-# Goal 3: Finish up [WIP](https://en.wikipedia.org/wiki/Work_in_process) features
+# 🎨 Goal 3: Survey customization
 
-We've started and nearly finished a couple of features our customers requested, but they are not [GA](https://en.wikipedia.org/wiki/Software_release_life_cycle#General_availability_(GA)) yet, the goal is to finish them up, gather feedback and put them in front of customers.
+A lot of people use the API to tweak their surveys. Let’s bring those use cases into the product.
 
-- [Adaptive Sampling](https://github.com/PostHog/posthog/issues/25722)
-    - Currently under a feature flag
-- [Partial Responses](https://github.com/PostHog/posthog/issues/19300)
-    - There are Draft PRs
-- [Survey Actions](https://github.com/PostHog/posthog/issues/27378)
-    - Currently under a feature flag
-- [AI Summaries](https://github.com/PostHog/posthog/issues/27379)
-    - Currently under early access features
+- [Better wait period controls](https://github.com/PostHog/posthog/issues/30379)
+- Alignment customization [here](https://github.com/PostHog/posthog/issues/28479) and [here](https://github.com/PostHog/posthog/issues/28477)
+- [Inline surveys](https://github.com/PostHog/posthog/issues/30097)
+- [Add icons](https://github.com/PostHog/posthog/issues/28478)
+- [Templated strings](https://github.com/PostHog/posthog/issues/26620)
+- [Single-response submit](https://github.com/PostHog/posthog/issues/30196)
 
-# Goal 4: Mobile Surveys
+# 🧩 Goal 4: Templates, Templates, Templates
 
-We want to bring the same functionality to Mobile that we have for Web Surveys, [GH Issue](https://github.com/PostHog/posthog/issues/25085).
+Right now, duplicating and customizing surveys across projects/environments isn’t great. Let’s fix that.
 
-- React Native
-- iOS
-- Android
-- Flutter
+- [Save custom templates](https://github.com/PostHog/posthog/issues/27309)
+- [Duplicate surveys across projects/environments](https://github.com/PostHog/posthog/issues/29138)
+- Add mobile-specific templates
+- Explore using AI to create survey templates
+
+# 💬 Goal 5: Make Key Customers Super Happy
+
+We’ve had some great convos with customers — now let’s turn those into lasting partnerships.
+
+- Build a case study with a power user
+- Be lightning-fast at resolving their issues
+
+# 🧑‍💻 Goal 6: Hire a Cracked Product Engineer
+
+The team’s growing! We’re hiring a Product Engineer to help us build faster and better.
+
+- Hire and onboard our next teammate
+- Make onboarding smooth and fast so they can hit the ground running
+
+# 🎯 Side Quests (Stretch Goals)
+
+We started these, but shifted priorities a bit. Still aiming to wrap them up if time allows.
+
+- [Adaptive survey sampling](https://github.com/PostHog/posthog/issues/25722)
+- [Survey actions](https://github.com/PostHog/posthog/issues/27378)
+- [Improve the Onboarding UX](https://github.com/PostHog/posthog/issues/30377)
```

Comment:
should we drop this altogether? we have plenty of goals and they have higher prior now
this could be a distraction, and if we have time, we can do this anyway!

Reply:
Yeah, I'd just make sure they are on the roadmap so people can vote on them.